library/pipebio_client.py

```diff
@@ -15,30 +14,35 @@
 
 class PipebioClient:
     _session: Session
-    authentication: Authentication
     shareables: Shareables
     entities: Entities
     jobs: Jobs
     sequences: Sequences
+    user: Any
 
     def __init__(self):
         self._session = Session()
+        api_key = os.environ['PIPE_API_KEY'] if 'PIPE_API_KEY' in os.environ else None
+        if api_key is None:
+            print('PIPE_API_KEY required.')
+            print(f'PIPE_EMAIL={api_key}')
```

Comment:
Something is wrong here, I assume there should be no PIPE_EMAIL required anymore and so we change it to: 
`PIPE_API_KEY={api_key}`?

Reply:
YES!